idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -485,66 +472,268 @@ def _make_performance(self):
+                     /blk.dp_wet[t, 1-x]))**0.5
+                    
+            self.gas_velocity_flood = Expression(self.flowsheet().time,
+                                    self.vapor_phase.length_domain,
+                                    rule=rule_flood_velocity,
+                                    doc='Gas velocity at flooding point')
+            
+        # Flooding fraction 
+        def rule_flood_fraction(blk, t, x):
+            return blk.velocity_vap[t, x]/blk.gas_velocity_flood[t, x]
+            
+        self.flood_fraction = Expression(self.flowsheet().time,
+                                    self.vapor_phase.length_domain,
+                                    rule=rule_flood_fraction,
+                                    doc='Flooding fraction (expected to be below 0.8)')
+        
+        # Mass and heat transfer terms
+        
+        # Mass transfer terms
+        self.pressure_equil = Var(
+            self.flowsheet().time,
+            self.vapor_phase.length_domain,
+            equilibrium_comp,
+            domain=NonNegativeReals,
+            initialize=500,
+            units=pyunits.Pa,
+            doc='Equilibruim pressure of diffusing components at interface')
+        
+        self.interphase_mass_transfer = Var(
+            self.flowsheet().time,
+            self.liquid_phase.length_domain,
+            liq_comp,
+            domain=Reals,
+            initialize=0.0,
+            units=pyunits.mol / (pyunits.s * pyunits.m),
+            doc='Moles of diffusing species transfered into liquid')
+        
+        self.enhancement_factor = Var(self.flowsheet().time,
                                       self.liquid_phase.length_domain,
                                       units=None,
-                                      domain=NonNegativeReals,
+                                      mutable=True,
                                       initialize=160,
                                       doc='Enhancement factor')
+        
+        self.enhancement_factor.fix(160)
 
-        self.yi_MEA = Var(self.t,
-                          self.liquid_phase.length_domain,
-                          domain=NonNegativeReals,
-                          initialize=0.5,
-                          units=None,
-                          doc='''Dimensionless concentration of MEA
-                                    at interface ''')
-        self.yeq_CO2 = Var(self.t,
-                           self.liquid_phase.length_domain,
-                           domain=NonNegativeReals,
-                           initialize=0.5,
-                           units=None,
-                           doc='''Dimensionless concentration of CO2
-                                      in equilibruim with the bulk''')
-
-        # heat transfer
-        self.heat_vap = Var(self.t,
+        # Heat transfer
+        self.heat_vap = Var(self.flowsheet().time,
```

Comment:
I think a more descriptive name is needed. This could be confused for heat of vapourisation.